proto/exocore/oracle/v1/native_token.proto

```diff
@@ -0,0 +1,53 @@
+syntax = "proto3";
+
+package exocore.oracle.v1;
+
+import "gogoproto/gogo.proto";
+
+option go_package = "github.com/ExocoreNetwork/exocore/x/oracle/types";
+
+// BalanceInfo tells effective-balance for native-restaking asset
+message BalanceInfo {
+  // current round of price, this is updated based on tokenFeeder
+  uint64 round_id = 1 [(gogoproto.customname)="RoundID"];
+  // on which block the balance updated, deposit/withdraw will update this and round_id might not change
+  uint64 block = 2;
+  // index used to distinguish multiple balanceInfo for one same roundid
+  uint64 index = 3;
+  // beaconchain epoch related to slash_refund
+  // uint epoch = 3;
+  int64 balance = 4;
+
+  // Action defines the reason the balance changed
+  enum Action {
```

Comment:
nit: move this outside `BalanceInfo` please. not a blocker.

Reply:
updated